README.md

```diff
@@ -37,6 +37,7 @@ resulting in the error. To fix this, you will need to run `docker logout` from y
 1. Run `cp ./.env.example ./.env`
 1. Run `cp ./apps/server/.env.example ./apps/server/.env`
 1. Follow [Initial Setup](#initial-setup) (below) for setting-up some required environment variables
+1. Install [ElasticSearch] via docker (https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) `docker pull elasticsearch:8.13.4`
```

Comment:
I'm pretty sure I didn't have to do this, but I did have to make sure I had Rancher Desktop installed

Reply:
Oh yeah I think you're right, the run command should pull the image if it doesn't exist. I'll remove this line.